File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.java
class Solution {
    public String solution(String rsp) {
        String result = "";
        for(int i=0; i<rsp.length(); i++){
            if(rsp.charAt(i) == '5') {
                result += '2';
            }else if(rsp.charAt(i) == '2') {
                result += '0';
            }else {
                result += '5';
            }
        }
        return result;
    }
}

File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.js
function solution(rsp) {
    let result = '';
    let ans = rsp.split('');
    for(let e of ans) {
        if(e === '5') {
            result += 2;
        }else if(e === '2') {
            result += 0;
        }else {
            result += 5;
        }
    }
    return result;
}
